lib/nexmo_developer/app/presenters/page_title.rb

```diff
@@ -6,14 +6,35 @@ def initialize(product, document_title)
 
   def title
     if @product && @document_title
-      "#{@product.titleize} > #{@document_title} | #{default_title}"
+      "#{normalized_product_title(@product)} > #{@document_title} | #{default_title}"
     elsif @document_title
       "#{@document_title} | #{default_title}"
     else
       default_title
     end
   end
 
+  def normalized_product_title(product)
+    if product.include?('/')
+      product_name = product.split('/')[0]
+      product_category = product.split('/')[1]
+      product_name_normalized = product_name.split(' ').map do |phrase|
+        phrase.split('-').map do |word|
+          (word.downcase == 'api') || (word.downcase == 'sdk') ? word.upcase : word.titleize
```

Comment:
Could we make this config driven? These rules work for the API group but other teams will need custom logic e.g.

```yaml
---
api: upcase
sdk: upcase
vbc: upcase
alwayslower: lowercase
```

Reply:
You know that's a really good idea. I'll refactor accordingly.

Reply:
@mheap, refactored to run as a config driven setting. I also updated the README and added a sample config for reference.